Kickstarter-iOS/Views/Controllers/ShippingRulesTableViewController.swift

```diff
@@ -50,8 +60,8 @@ final class ShippingRulesTableViewController: UITableViewController {
     self.viewModel.outputs.deselectCellAtIndex
       .map { IndexPath(row: $0, section: 0) }
       .observeForUI()
-      .observeValues { [weak self] indexPath in
-        self?.tableView.cellForRow(at: indexPath)?.accessoryType = .none
+      .observeValues { [weak self] _ in
+        self?.tableView.visibleCells.forEach { $0.accessoryType = .none }
```

Comment:
This simplified a lot of logic, because we don't have to keep in track previously selected index (this gets really complicated in case the list is filtered causing that previous index to be out of sync).

To simplify this behaviour we simply make sure to deselect cells first, then selecting the latest index.